### DISCUSSION TITLE
Prevent Non-Admin Users from Modifying Global Theme Setting

### DISCUSSION THREAD

**Discussion by SexyWerewolf** [crocodilestick/Calibre-Web-Automated#136] [Ideas]
### Proposal: Restrict Non-Admin Users from Changing 'cwa-switch-theme' Globally

This feature would prevent non-admin users from modifying the global `cwa-switch-theme` setting. Currently, all users can change the theme, which may lead to inconsistencies. Limiting this to admins only ensures a consistent user experience across the application.

**Comment by crocodilestick** [comment]
Hey @SexyWerewolf , yeah that makes sense, I'll look into it for the next version

**Comment by jspiers** [comment]
I'd also like to see this. It's a bit weird at the moment... any user can change the theme and it affects the UI for all the other users. And then another user can switch it back. Theme wars. :)
